Lasse/LittleWindows.asm

```diff
@@ -166,12 +168,10 @@ find_egg:
         inc edi                                 ; increase address by 1
         cmp dword ptr ds:[edi], eax             ; check for "DAVE"
         jne find_egg                            ; loop if not found
-        add edi, 4                              ; move to next 4 bytes
-        cmp dword ptr ds:[edi], eax             ; check for "DAVE" again
+        cmp dword ptr ds:[edi + 4], eax         ; check for "DAVE" again
         jne find_egg                            ; loop if not found
 matched:
-        mov ebx, edi                            ; put the place we found in ebx...
-        sub ebx, sp_egg + 4                     ; ...and adjust it
+        lea ebx,[edi - sp_egg]                  ; return the adjusted place we found

```

Comment:
Please add a space after the comma.

Reply:
Did so, I think/hope this was the last formatting error